app/src/forms/form/routes.js

```diff
@@ -54,7 +53,7 @@
   await controller.deleteForm(req, res, next);
 });
 
-routes.get('/:formId/submissions', middleware.publicRateLimiter, apiAccess, hasFormPermissions([P.FORM_READ, P.SUBMISSION_READ]), async (req, res, next) => {
+routes.get('/:formId/submissions', apiAccess, hasFormPermissions([P.FORM_READ, P.SUBMISSION_READ]), async (req, res, next) => {
```

Comment:
One of two things is going on here:
1. CodeQL isn't smart enough to see the rate limiting applied to the app as a whole
2. CodeQL is smart enough to see that the rate limiting is only applied to some callers

Guessing it's option 1 - should we apply it to every endpoint to make CodeQL happy?

Reply:
Hmm, actually since the global limiter set in app.js reads from a config file, the actual path isn't know until runtime. So the static checker can't tell that the endpoints are being covered.